plugins/browser-plugin-snowplow-ecommerce/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@snowplow/browser-plugin-snowplow-ecommerce",
+  "version": "3.7.0",
```

Comment:
Does the version need to be bumped? Or is that done in the release?

Reply:
This is done during the release command :)